index.js

```diff
@@ -13,12 +16,15 @@ async function generateBot(token, name) {
     },
     data: {
       name,
+      bot_public: false,
```

Comment:
```suggestion
      bot_public: true,
```

Reply:
The reason why should be true, is premium bots can be invited by others instead of account owner

Reply:
i don't think so 
it was working with false

Reply:
Cuz you're the owner of the bot